packages/cannon-worker-api/src/cannon-worker-api.ts

```diff
@@ -42,6 +42,14 @@ export class CannonWorkerAPI extends EventEmitter {
     this.postMessage({ op: 'setGravity', props: value })
   }
 
+  get frictionGravity(): Triplet | undefined {
+    return this.config.frictionGravity;
+  }
+  set frictionGravity(value: Triplet | undefined) {
```

Comment:
It's not an optional value here.

Also, could you please keep the ordering of the properties alphabetical, this should go above gravity, I'm not sure why our linter is not complaining about it.

Reply:
Ah, the linter didn't run.

Reply:
You got it!

Reply:
Could you clarify what you mean by it not being optional here? My intention here was to get across that the user may define `frictionGravity`, and they may also reset it back to undefined.